Sources/KituraContracts/ClousureAliases.swift

```diff
@@ -18,49 +18,73 @@
 
 /**
 The `ResultClosure` is used by other `Codable` aliases when responding with only a `RequestError` is needed
-*/
+ 
+ ````swift
+ // The following two typealias make use of ResultClosure
```

Comment:
Reword to - "The following two typealiases make use of `ResultClosure`:" and move this out of the code section. Should this be in Example Usage? or is this intended just for informational purposes?